cleverhans_tutorials/mnist_tutorial_jsma.py

```diff
@@ -62,7 +62,7 @@ def mnist_tutorial_jsma(train_start=0, train_end=60000, test_start=0,
     set_log_level(logging.DEBUG)
 
     # Get MNIST test data
-    X_train, Y_train, X_test, Y_test = data_mnist(train_start=train_start,
+    x_train, y_train, x_test, y_test = data_mnist(train_start=train_start,
```

Comment:
Why did you change the variable names?

Reply:
To be PEP8 compliant, I just did it mechanically since I was touching the file,

Reply:
Thanks!